nucliadb_vectors/src/service/writer.rs

```diff
@@ -56,26 +57,57 @@ impl Debug for VectorWriterService {
     }
 }
 
+struct DummyPreparedMerge();
```

Comment:
You dont need to use `()` in Rust, a zero-size type will do:
`struct DummyPrepared;`